### PR TITLE
cassandra 5 single tests

### PR DESCRIPTION
**Pull request by conorbros**
pending:
- [x] https://github.com/shotover/cassandra-test-images/pull/5 being merged will mean I can reenable the function tests
- [x] fix batch size error, seems cassandra 5 wants lower batch sizes

### CODE REVIEW

**Comment by rukai** [comment]
Lets land the absolute bare minimum first.
vectors and other cassandra 5 features can come later.
I'll review https://github.com/shotover/cassandra-test-images/pull/5/files asap

**Comment by rukai** [comment]
https://hub.docker.com/layers/shotover/cassandra-test/5.0-beta1-r2/images/sha256-261210443b941dc0fd9a0347cf43637c41275dfa490ff8bab072dd8777145e46?context=explore the new image is now available

**Comment by conorbros** [comment]
Only Java user defined functions are supported in Cassandra 5. Lets merge this with the bare minimum and I'll open another PR for testing UDF in Cassandra 5.

File: shotover-proxy/tests/cassandra_int_tests/batch_statements.rs [hunk 1]
```diff
@@ -11,7 +11,7 @@ async fn large_batch(connection: &CassandraConnection) {
     let random_bytes = rand::thread_rng().gen::<[u8; 32]>();
 
     let mut queries = Vec::new();
-    for id in 0..1000 {
+    for id in 0..500 {
```
Comment (rukai) [inline shotover-proxy/tests/cassandra_int_tests/batch_statements.rs:14]: what is this for?

Comment (conorbros) [reply shotover-proxy/tests/cassandra_int_tests/batch_statements.rs:14]: >fix batch size error, seems cassandra 5 wants lower batch sizes

**Review by cjrolo — APPROVED** [review]

**Pull request merged**